api/internal/utils/timedcall_test.go

```diff
@@ -62,3 +63,15 @@ func TestTimedCallSlowWithError(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestTimedCallGoroutineLeak(t *testing.T) {
+	beforeGroutine := runtime.NumGoroutine()
```

Comment:
Small typo in variable name:
```suggestion
	beforeGoroutine := runtime.NumGoroutine()
```